spec/tunes/build_spec.rb

```diff
@@ -42,12 +42,17 @@
     end
 
     describe "#testing_status" do
+      before do
+        now = Time.at((1_443_144_470 + 1_445_737_214) / 2)
```

Comment:
Do not use `Time.at` without zone. Use one of `Time.zone.at`, `Time.at.current`, `Time.at.in_time_zone`, `Time.at.utc`, `Time.at.getlocal`, `Time.at.iso8601`, `Time.at.jisx0301`, `Time.at.rfc3339`, `Time.at.to_i`, `Time.at.to_f` instead.